### PR TITLE
Payment system

### PR DESCRIPTION
**Pull request by kathrynowy**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -7,6 +7,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
   <meta name="theme-color" content="#000000" />
   <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+  <script src="https://js.stripe.com/v3/"></script>
```
Comment (dzmitry-penda) [inline public/index.html]: Do you really need this if u already added stripe to npm packages list?

File: src/Components/CheckoutForm/CheckForm.scss [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+.Checkout {
+  margin: 0 auto;
+  max-width: 800px;
+  box-sizing: border-box;
+  padding: 0 5px;
+  background-color: $payment-form;
+}
+
+label {
+  color: $payment-forrm-label;
```
Comment (dzmitry-penda) [inline src/Components/CheckoutForm/CheckForm.scss]: forrrrrrrrrrrrm

File: src/Components/CheckoutForm/CheckoutForm.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import { CardElement, injectStripe } from 'react-stripe-elements';
+import './CheckForm.scss';
+
+class CheckoutForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { complete: false, isSubmit: props.isSubmit };
```
Comment (dzmitry-penda) [inline src/Components/CheckoutForm/CheckoutForm.js]: state.complete is unused

File: src/Components/Hall/ConfirmTicket/ConfirmTicket.js [hunk 4]
```diff
@@ -52,12 +68,20 @@ class ConfirmTicket extends Component {
     this.props.buyTickets(confirmedTickets);
   }
 
+  async buyConfirmedTickets(token, url, totalCost) {
+    let { data } = await axios.post("http://localhost:8080/payment", { token, totalCost });
```
Comment (dzmitry-penda) [inline src/Components/Hall/ConfirmTicket/ConfirmTicket.js]: You should use base url

File: src/Components/PaymentModal/PaymentModal.js [hunk 5]
```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
```
Comment (dzmitry-penda) [inline src/Components/PaymentModal/PaymentModal.js]: Combine imports into one

File: src/Components/PaymentModal/PaymentModal.js [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import { Elements } from 'react-stripe-elements';
+import CheckoutForm from '../CheckoutForm/CheckoutForm'
+
+
+class PaymentModal extends React.Component {
+  state = {
+    open: false,
```
Comment (dzmitry-penda) [inline src/Components/PaymentModal/PaymentModal.js]: isOpen

File: src/index.js [hunk 7]
```diff
@@ -16,8 +16,10 @@ axios.defaults.baseURL = apiBaseUrl;
 
 
 ReactDOM.render(
-  <Provider store={store}>
-    <App />
-  </Provider>,
+  <StripeProvider apiKey="pk_test_A8lmf6gBkwNhNx21XyRWQQXH00QQGcbT1R">
```
Comment (dzmitry-penda) [inline src/index.js]: This should be moved to the config. By the way, never put any keys pn the github

File: public/index.html (hunk 1 above)
Comment (kathrynowy) [reply public/index.html]: Yes, i do

**Pull request merged**